src/mscorlib/src/System/AppDomain.cs

```diff
@@ -837,10 +837,17 @@ public Object GetData(string name)
                 }
             }
         }
-        
+       
+#if !PLATFORM_UNIX        
```

Comment:
I would have to #if !UNIX several pieces of EventSource. I can do that if you prefer.